### PR TITLE
add notation for boundary of a set of dimensions??

### PR DESCRIPTION
**Pull request by jonsterling**
While we're golfing code, what do folks think about this notation?

/cc @mortberg @cangiuli @ecavallo @favonia

### CODE REVIEW

**Comment by favonia** [comment]
I wish there was an ASCII version, but could not find a good symbol.

**Comment by jonsterling** [comment]
@favonia Yeah, I too was trying to figure out what would be a reasonable ascii version... :frowning:

File: vim/syntax/redtt.vim [hunk 1]
```diff
@@ -22,7 +22,7 @@ syn region  redttEncl transparent start="\[" end="\]" contains=ALLBUT,redttBrack
 
 syn match   redttHole '?\k*'
 
-syn keyword redttKeyw data intro where in with end elim
+syn keyword redttKeyw boundary data intro where in with end elim
```
Comment (jonsterling) [inline vim/syntax/redtt.vim]: is this intended?

Comment (favonia) [reply vim/syntax/redtt.vim]: No. LOL

**Comment by favonia** [comment]
@jonsterling I suppose we are both okay with the new syntax (?).

**Pull request merged**